### PR TITLE
Fix issue#2586

### PR DESCRIPTION
**Pull request by Wetinee**
Add 'upload' part to make NzI18nService support fr_FR.

### CODE REVIEW

File: components/i18n/languages/fr_FR.ts [hunk 1]
```diff
@@ -14,6 +14,8 @@ export default {
     filterConfirm: 'OK',
     filterReset: 'Réinitialiser',
     emptyText: 'Aucune donnée',
+      selectAll: 'tout sélectionner',
```
Comment (wendzhue) [inline components/i18n/languages/fr_FR.ts]: Indent

Comment (Wetinee) [reply components/i18n/languages/fr_FR.ts]: OK and thanks for your advice.

Comment (wendzhue) [reply components/i18n/languages/fr_FR.ts]: Oh. And the first character should be uppercase. Didn't noticed that. Sorry. 🤣

Comment (Wetinee) [reply components/i18n/languages/fr_FR.ts]: My mistake. See if it's ok now.

**Comment by wendzhue** [comment]
Thanks for submitting this pr. Related problem was closed by https://github.com/NG-ZORRO/ng-zorro-antd/pull/2737.

**Pull request closed without merging**